jsx/Form.js

```diff
@@ -10,6 +10,8 @@ NumericElement, FileElement, StaticElement, LinkElement, ButtonElement, LorisEle
  *
  */
 
+import React, {Component, PropTypes} from 'react';
```

Comment:
not sure if this is needed as React is already imported elsewhere in LORIS

Reply:
@xlecours i am curious what you think about `import PropTypes from 'prop-types'` library vs. calling React.PropTypes for every prop

Reply:
@zaliqarosli It is the right thing to do if we want to use the React library from npm. 

@HenriRabalais Don't you endup using 2 version of React on the same page since NDB_Page adds it in getJSDependencise? It think we should remove ou "in-house" version of react and rely on npm but we might have to change most of our jsx files and retranspile the components of LORIS at large

Reply:
@xlecours So you suggest that I keep `import React from 'react'` at the top of the page and we remove it from the NDB_Page getJSDependencies?

Reply:
Yes.
But it will involve a lot of work. (rewriting all the data query tool app to use React instead of the htdocs/js/react/react.js since it won't be available throught the page. Or, add htdocs/js/react/react.js to all getJSDependencies function of "old" modules

Reply:
Okay, in that case it may be best to remove these changes from this pull request and create a new pull request for this purpose.